combine.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Takes two SVG files (the first two arguments) and layers them above each other.
+# The third argument is the output file.
+# Make sure that the canvas size is the same in all files,
+# otherwise the positioning of the images is not correct!
+
+set -e
+
+if [[ $# -ne 3 ]]; then
+    echo "ERROR! Please provide exactly 3 arguments (two input files and one output file)."
+    exit 1
+fi
+
+# TODO Check canvas sizes
+
+# Make sure all input files went through svgo for homogenous structure
+for file in "$1" "$2"; do
+    svgo --pretty --indent 0 --final-newline "$file"
```

Comment:
Would be good to have a check to ensure SVGO is available as in `build.sh`.
Also could it output a usage to explain the options (basically extend line 11's echo)?

Reply:
> Would be good to have a check to ensure SVGO is available as in `build.sh`.

I thought about this as well, but since I am calling `combine.sh` from `build.sh`, this check already happened. But you are right - someone might run `combine.sh` directly. I'll address that later today!